Assets/MixedRealityToolkit.Services/DiagnosticsSystem/MixedRealityToolkitVisualProfiler.cs

```diff
@@ -53,6 +54,12 @@ public bool IsVisible
             set { isVisible = value; }
         }
 
+        private bool ShouldShowProfiler => isVisible
+#if WINDOWS_UWP
+            && (appCapture == null || !appCapture.IsCapturingVideo)
+#endif // WINDOWS_UWP
+            ;
```

Comment:
This training semicolon feels like it could be dangerous... I think it's the way of saving the most vertical space but it sticks out so much I wonder if it would be better just to have a UWP version and non UWP version, or just put the WINDOWS_UWP part first i.e. 

```csharp
 private bool ShouldShowProfiler =>
#if WINDOWS_UWP
            (appCapture == null || !appCapture.IsCapturingVideo) && 
#endif // WINDOWS_UWP
             isVisible ;
```

Reply:
I like the `WINDOWS_UWP` part first

Reply:
Updated